lib/ClangImporter/ImportEnumInfo.cpp

```diff
@@ -308,6 +307,8 @@ void EnumInfo::determineConstantNamePrefix(ASTContext &ctx,
     // Don't use importFullName() here, we want to ignore the swift_name
     // and swift_private attributes.
     StringRef enumNameStr = decl->getName();
+    if (enumNameStr.empty())
+      enumNameStr = decl->getTypedefNameForAnonDecl()->getName();
```

Comment:
Assert / error that it now has a name?

Reply:
Ah, yeah, that's how it got caught originally, but I guess it'd be nicer to catch it here than in `getCommonPluralPrefix`.